integration_tests/test_docs.py

```diff
@@ -56,11 +57,17 @@ def integration_test_user_id() -> str:
 
 
 def modify_source(source: str) -> str:
-    # Adds some custom logic to update code from docs to comply with some requirements.
-    source = re.sub(r"('team'|\"team\"): ('\w+'|\"\w+\")", r"'team': 'infra'", source)
+    """Adds some custom logic to update code from docs to comply with some requirements."""
```

Comment:
nit: docstring is super vague, could we clarify

Reply:
updated comments in the fn to explain actions